hive-tables-creation.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+
+client = boto3.client('glue')
+response = client.create_table(
+    DatabaseName='analytical_dataset_generation',
+    TableInput={
+        'Name': 'core_contract_adg',
+        'Description': 'core_contract_adg',
+        'StorageDescriptor': {
+            'Columns': [
+                {
+                    'Name': 'rowkey',
+                    'Type': 'string'
+                },
+                {
+                    'Name': 'data',
+                    'Type': 'string'
+                }
+            ],
+            'Location': 's3://{REPLACE_THIS}/hive/external/core_contract_adg',
```

Comment:
Again, remove the `adg` suffix please. Hmm, perhaps we need to namespace this a bit differently too. How does this sound?

`s3://{REPLACE_THIS}/analytical-dataset/hive/core/contract' ?

When we get to it, then the PDM would go under:

`s3://{REPLACE_THIS}/pdm/hive/contract_dim`

Reply:
Changed as suggested